packages/lib/services/share/reducer.ts

```diff
@@ -82,10 +82,6 @@ export const parseShareCache = (serialized: string): State => {
 	};
 };
 
-export const readFromSettings = (state: RootState): State => {
-	return parseShareCache(state.settings['sync.shareCache']);
-};
```

Comment:
`readFromSettings` was added in https://github.com/laurent22/joplin/commit/77482a0c956987434a4e1f7d645aeec8d51292eb#diff-abe2fce8500ff5467a22c4d5983313ee7f3d2085895706702a57aa7f8274e3edR85 and is now unused.